### PR TITLE
fix(pill-selector-dropdown): adds padding back for scrollable dropdowns

### PR DESCRIPTION
**Pull request by danubevictoria**
Fixes padding on top of overlay for pill selector dropdowns and dropdowns in USM.

Before:
image

After:
image

with Suggested Collabs:
image

with only one result:
image

styleguidist:
image

styleguidist pill selector dropdown with `overlayTitle="suggested"` prop for suggested users:
image

### CODE REVIEW

**Comment by booc0mtaco** [comment]
@danubevictoria can you confirm the spacing when there is only one item in the result list?

File: src/components/selector-dropdown/SelectorDropdown.js [hunk 1]
```diff
@@ -256,7 +256,7 @@ class SelectorDropdown extends React.Component<Props, State> {
         }
 
         const list = (
-            <ul className="overlay" id={listboxID} role="listbox">
+            <ul className={`overlay ${overlayTitle ? overlayTitle.toLowerCase() : ''}`} id={listboxID} role="listbox">
```
Comment (booc0mtaco) [inline src/components/selector-dropdown/SelectorDropdown.js:259]: can you use `classNames` library here? also, where does overlayTitle come from?

Comment (danubevictoria) [reply src/components/selector-dropdown/SelectorDropdown.js:259]: will use classNames. overlayTitle comes from PillSelectorDropdown as a prop, which is passed in from ContactSelector in EUA.

File: src/components/selector-dropdown/SelectorDropdown.js [hunk 2]
```diff
@@ -256,7 +256,11 @@ class SelectorDropdown extends React.Component<Props, State> {
         }
 
         const list = (
-            <ul className="overlay" id={listboxID} role="listbox">
+            <ul
+                className={classNames('overlay', overlayTitle ? overlayTitle.toLowerCase() : '')}
```
Comment (booc0mtaco) [inline src/components/selector-dropdown/SelectorDropdown.js:260]: See [docs](https://www.npmjs.com/package/classnames). you dont wanna include ternary operations when using this function

Comment (danubevictoria) [reply src/components/selector-dropdown/SelectorDropdown.js:260]: since overlayTitle is optional we discussed that the ternary is ok.

**Pull request merged**